solidity/deploy/07_deploy_bridge_governance.ts

```diff
@@ -0,0 +1,44 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // TODO: validate proxy bridge vs bridge implementation
+  const Bridge = await deployments.get("Bridge")
+  const BridgeGovernanceParameters = await deployments.deploy(
+    "BridgeGovernanceParameters",
+    {
+      from: deployer,
+      log: true,
+    }
+  )
+
+  const GOVERNANCE_DELAY = 604800 // 1 week
+
+  const BridgeGovernance = await deploy("BridgeGovernance", {
+    contract:
+      deployments.getNetworkName() === "hardhat"
+        ? "BridgeGovernance"
+        : undefined,
```

Comment:
We don't need this. We use this only when we deploy a stub.
```suggestion
```

Reply:
[00a2f65](https://github.com/keep-network/tbtc-v2/pull/329/commits/00a2f650f4c05f542c65d253e95a0f6c2317a34a)